src/components/Project/project.scss

```diff
@@ -50,6 +50,79 @@
         margin-left: 10px;
       }
 
+      &__source {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        min-height: 26px;
+
+        .input-wrapper {
+          width: 100%;
+        }
+
+        .input {
```

Comment:
you have input declaration, we are using SCSS, don't duplicate ( see input-wrapper )